jwst/regtest/test_nircam_image.py

```diff
@@ -72,6 +72,19 @@ def run_image3pipeline(run_image2pipeline, rtdata_module, jail):
     Step.from_cmdline(args)
 
 
+@pytest.fixture(scope="module")
```

Comment:
If you scope it like this, this data will be written to the same tmpdir as the other dataset in this module.  Probably not what we want.  Probably better to make this function-scoped and and use the `rtdata` fixture instead of the `rtdata_module` fixture since there is only one test function that calls this fixture.